src/main/resources/org/openbase/bco/bcozy/controller/powerterminal/PowerTerminalSidebarPane.fxml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?import javafx.scene.layout.*?>
+
+<?import com.jfoenix.controls.JFXComboBox?>
+<?import com.jfoenix.controls.JFXCheckBox?>
+<?import javafx.scene.text.Text?>
+<?import com.jfoenix.controls.JFXDatePicker?>
+<VBox xmlns="http://javafx.com/javafx"
+      xmlns:fx="http://javafx.com/fxml"
+      fx:controller="org.openbase.bco.bcozy.controller.powerterminal.PowerTerminalSidebarPaneController"
+      alignment="TOP_CENTER"
+      spacing="40">
+
+    <JFXComboBox fx:id="selectVisualizationTypeBox" promptText="Visualization" prefWidth="200"/>
+    <JFXComboBox fx:id="selectUnitBox" promptText="Unit" prefWidth="200"/>
+    <JFXComboBox fx:id="selectGranularityBox" promptText="Granularity" prefWidth="200"/>
+    <VBox spacing="5" prefWidth="200" alignment="TOP_CENTER">
+        <HBox spacing="5" alignment="TOP_CENTER">
+            <JFXCheckBox fx:id="selectDateNowCheckBox"/>
+            <Text text="Current Consumption"/>
+        </HBox>
+        <JFXDatePicker fx:id="selectStartDatePicker" defaultColor="#303034" promptText="Start Date" prefWidth="200"/>
+        <JFXDatePicker fx:id="selectEndDatePicker" defaultColor="#303034" promptText="End Date" prefWidth="200"/>
+        <Text fx:id="dateErrorMessage" fill="red" text="Error: Date invalid!"/>
```

Comment:
same over here.

Reply:
Never displayed Prompt texts removed, error message yet needs to be localized

Reply:
Text localized in 3b17509